src/Appear.js

```diff
@@ -29,6 +29,10 @@ export default withDeck(class Appear extends React.Component {
       return children
     }
 
+    if (window.navigator.userAgent === 'pdf') {
```

Comment:
I think using a spec compliant user-agent would be great. Maybe smth. like `MDX-Deck/1.7.7 Print/PDF`?

This way later plugins / components have an easier time differentiating if needed. And your check would just be `window.navigator.userAgent.includes('Print/PDF')`

Reply:
I totally agree.